app.js

```diff
@@ -0,0 +1,135 @@
+require("dotenv").config();
+const express            = require('express');
+const path               = require('path');
+const favicon            = require('serve-favicon');
+const logger             = require('morgan');
+const cookieParser       = require('cookie-parser');
+const bodyParser         = require('body-parser');
+const expressLayouts     = require('express-ejs-layouts');
+const passport           = require('passport');
+const LocalStrategy      = require('passport-local').Strategy;
+const User               = require('./models/user');
+const bcrypt             = require('bcrypt');
+const session            = require('express-session');
+const MongoStore         = require('connect-mongo')(session);
+const mongoose           = require('mongoose');
+const flash              = require('connect-flash');
+var multer  = require('multer');
+var upload = multer({ dest: './public/uploads/' });
+
+
+mongoose.connect(process.env.MONGODB_URI);
```

Comment:
You could probably write something like 
mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost:27017/tumblr-lab-development');
Then in case of no env variable, it will fallback to local server for development